### DISCUSSION TITLE
2FA: GitHub always asks for the same two digits

### DISCUSSION THREAD

**Discussion by nkehrein** [community/community#60478] [Mobile]
### Select Topic Area

Bug

### Body

I am using the GitHub app on my mobile for 2FA. The websites asks me to authenticate using 2FA from time to time when doing various repo admin tasks. It will show a popup with two digits which I have to enter into an app dialog. The thing is, it is always the same two digits. This has to be a bug, doesn't it? Or is Github following the XKCD approach to random numbers?

![image](https://github.com/community/community/assets/74192425/4f6b6e77-9cca-4e95-8794-d2f517054713)

**Comment by taki-on** [comment]
Thank you very much for sharing the issue with us! It is odd that you always get the same digits 😮 . I also use the mobile apps for 2FA and I get different digits every time 🤔 . I will file an issue and will see if we have anything odd in our code. Thank you again!

**Comment by nkehrein** [comment]
Thanks! I don't know if it's any help but the issue has not appeared again so far. The previous digits stayed constant for several week, though.

**Comment by mosheduminer** [comment]
I've noticed this recently. I often log in using the github CLI (I need to do so because I have two different accounts and the CLI does not support profiles), and I often get the same two digits. I haven't been tracking it, but I think it's been the same every time I've logged in for the past while.
